custom-login/src/Login.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import * as OktaSignIn from '@okta/okta-signin-widget';
+import { Container } from 'semantic-ui-react';
+import '@okta/okta-signin-widget/dist/css/okta-sign-in.min.css';
+import '@okta/okta-signin-widget/dist/css/okta-theme.css';
+
+import Navbar from './Navbar';
+import config from './.samples.config';
+
+export default class LoginPage extends Component {
+  constructor(props) {
+    super(props);
+    this.signIn = new OktaSignIn({
+      baseUrl: config.oktaSample.widget.baseUrl,
+      clientId: config.oktaSample.oidc.clientId,
+      redirectUri: config.oktaSample.oidc.redirectUri,
+      logo: '/react.svg',
+      i18n: {
+        en: {
+          'primaryauth.title': 'Sign in to React & Company',
+        },
+      },
+      authParams: {
+        responseType: ['id_token', 'token'],
```

Comment:
These values should be configured by default by `okta-react`. It would look cleaner if we omitted them.

Reply:
They are defaults when using `okta-react`, but in this case we are configuring the SIW and this configuration is required if you want to get an ID token back.